src/utils/gameStateUtil.ts

```diff
@@ -25,7 +26,14 @@ export const gameStateUtil = {
 
     return gameState;
   },
-  save: (state: GameState): void => {
-    databaseUtil.save(state);
+  save: (state: object): void => {
+    if (isGameState(state)) {
+      databaseUtil.save(state);
+      return;
+    }
+
+    throw new DeveloperError(
+      'Attempted to save an invalid game state. Did you forget to compile the game state schema?',
```

Comment:
Should we add the `state` they are trying to save here?

Reply:
I can't decide if I want to console.log it or stringify it and write to a file. So i'm going to defer to whoever gets this error first.

Reply:
It will be a race.

Reply:
🐀